rdtools/test/analysis_chains_test.py

```diff
@@ -743,6 +743,23 @@ def test_plot_degradation_timeseries(sensor_analysis, clearsky_analysis):
     )
 
 
+def test_energy_from_power_hourly_data():
+
+    times = pd.date_range("2019-01-01 00:00:00", periods=3, freq="H")
```

Comment:
`H` is deprecated, let's change to `h`
```suggestion
    times = pd.date_range("2019-01-01 00:00:00", periods=3, freq="h")
```

Reply:
@mdeceglie  - Thanks for pointing those out. I've changed them to lowercase.